aws/policy/security-services.yaml

```diff
@@ -138,6 +138,14 @@ Statement:
       - 'arn:aws:iam::{{ aws_account_id }}:role/ansible-test-*'
       # This is hard coded into DMS...
       - 'arn:aws:iam::{{ aws_account_id }}:role/dms-vpc-role'
+  - Sid: AllowEnhancedMonitoringPolicyAttach
```

Comment:
Apologies @marknet15 I wasn't able to finish testing this today.  I believe you need to add 'arn:aws:iam::aws:policy/service-role/AmazonRDSEnhancedMonitoringRole' around line 22 instead of this section.  However because the rds test suite takes so long to run I wasn't able to get a successful test run.  I'll keep working on this review.

So far when running it against the resources created by the test suite the terminator class seems good

Reply:
Hm, maybe not - just caught this on my latest test run.  Will need to debug more.
`"msg": "Unable to modify DB instance: An error occurred (InvalidParameterValue) when calling the ModifyDBInstance operation: IAM role ARN value is invalid or does not include the required permissions for: ENHANCED_MONITORING"`

Reply:
@jillr No I think you are correct ! I totally missed that section on policies ! I've removed my blob and added the policy to the allowed list. I think thats the only actual missing bit.

Reply:
Actually I think I know whats happening, I spotted something similar internally but I think there's some eventual consistency delay, when creating the IAM role. After a retry it seems to pass once the IAM role has fully provisioned..

Reply:
I'm best sure how to handle it as it seems intermittent, but I removed the IAM deletion at the end of the tests, as it's not cost incurring and wouldn't change.

Reply:
@marknet15 - 'terminator' (the python code living in this module) will delete any roles you leave lying around for too long.  If you're having problems with IAM roles being 'eventually consistent', I find it helps to move the iam role creation towards to the start of the tests.  Given how slow the RDS tests are this should give enough time for everything to settle down.

Reply:
@tremble thanks🙏 yep that’s a good suggestion, I moved the IAM creation to the start earlier, so fingers crossed that sorts it out as I think the issue was eventual consistency.